src/layout/intro.html

```diff
@@ -0,0 +1,146 @@
+<div class="home-page-intro">
+    <hr />
+    <h2>Workshop Agenda</h2>
+
+    <table class="agenda-table">
```

Comment:
How do you feel about https://github.com/helpers/helper-markdown#usage-examples?

Reply:
I could just be doing the most.

Reply:
I was looking into markdown, and it would be helpful in each of the code-step instructions for build and predict pages to provide more accessible content. I don't currently have any code examples in the homepage so it might not be necessary for the homepage?